terra/models.py

```diff
@@ -143,14 +144,15 @@ def international(self):
         return self.activity.country != "USA"
 
     def approved(self):
+        # TODO: Clarify how international approval trumps funded approval
         if self.international():
             return len(self.approval_set.filter(type="I")) == 1
-        return len(self.approval_set.filter(type="S")) == 1
+        return self.funded()
```

Comment:
The `approved()` was intended as a check on whether or not the supervisor/unit manager approved. And that is distinct from whether or not the AUL has approved the funding, which is why `funded()` has it's own method (and also why I went with `Funding` instead of `Approval` for the new bridge table). I guess this is an argument against using the term "Approved" in place of "Allocation". Perhaps we should say "Funded" instead.

Reply:
@joshuago78 So should I revise the `approved()` method to look only at approval types again, confirming (S)upervisor approval(s) exist?  I'm not sure if you're asking for more than that.

Reply:
The `TravelRequest` model also needs new fields, now that `Approval` is a bridge table about funding only.
- `approved_by` for the supervisor that approved the leave
- `approved_on` for the timestamp of the above approval
- `international_approved_on` for the timestamp of Ginny's approval (this will be null for most rows. We either live with that or create a new table just for international approvals.

Reply:
Fields added.  I used `DateField` instead of `DateTimeField` because they really aren't timestamps: it requires user action to set these, and I think we don't need time-specific granularity.

I also updated the `approved()` function to remove the now-obsolete approval type filters, and check the `approved_on` value(s).